geometry_msgs/Twist base_twist
float64[] joint_velocities

int64 SIZE = 11